.dockerignore

```diff
@@ -15,3 +15,4 @@
 !README.md
 !licenses/**
 !LICENSE
+!MANIFEST.in
```

Comment:
Will this make it so that we can't run mypy in a docker container?

Reply:
Here I was trying to bring MANIFEST.in into the container for `package_data` / `py.typed` related reasons - I've been doing all my dev in Docker, so getting mypy working in there is a top priority for me.